akka-sse/src/main/scala/de/heikoseeberger/akkasse/EventStreamMarshalling.scala

```diff
@@ -38,7 +38,7 @@ object EventStreamMarshalling extends EventStreamMarshalling
 trait EventStreamMarshalling {
 
   implicit final def toResponseMarshaller(implicit ec: ExecutionContext): ToResponseMarshaller[Source[ServerSentEvent, Any]] =
-    Marshaller.withFixedCharset(MediaTypes.`text/event-stream`, HttpCharsets.`UTF-8`) { messages =>
+    Marshaller.withFixedContentType(ContentTypes.`text/event-stream`) { messages =>
```

Comment:
I've declared `de.heikoseeberger.akkasse.ContentTypes` which is in the same package as `EventStreamMarshalling`, so it should compile:

https://github.com/hseeberger/akka-sse/pull/58/files#diff-1aa5312badff1f7a1dac1ab07dd7a5edR22